### ISSUE TITLE
Tableau instructions appear to be outdated for MacOS

### ISSUE THREAD

**Issue by erikhuck** [duckdb/duckdb-web#925]
I was not able to connect Tableau to DuckDB on my Mac using [the instructions](https://duckdb.org/docs/guides/data_viewers/tableau.html). This could be because I'm using an updated version of Tableau and perhaps the DuckDB documentation is for an outdated version. I am using Tableau version 2023.2.

I was able to connect using a modified version of the steps. I'd be willing to provide those steps if others have also had trouble connecting on a Mac and if so, the DuckDB documentation should be updated.

**Comment by hawkfish** [comment]
Sorry, yes, they are very outdated. These days I strongly recommend using [my connector](https://github.com/hawkfish/duckdb-taco). I have used it successfully with MacOS and a Linux Tableau Server. Please let me know if you have any problems with it.

**Comment by erikhuck** [comment]
@hawkfish I'm afraid that solution there was recommended to me before on a Tableau forum and did not work for my Macbook pro. They also require additional steps like running in development mode, which is cumbersome. I think the instructions in the Tableau documentation for using a JDBC are almost correct but the file location for the JDBC driver didn't work for me. I think if we simply tweak the instructions that already exist in the official DuckDB docs, I believe that would be a better solution than developing an entire other package just for this simple use case. Do you have access to the Tableau docs or are you in contact with those who have access?

**Comment by hawkfish** [comment]
@erikhuck Yes [I have access](https://duckdblabs.com) and I [submitted an update](https://github.com/duckdb/duckdb-web/pull/926).  (I'm also the guy who developed the Tableau connectivity APIs when I was there from 2005-2019, so I have some idea how to make it all work and where the pitfalls are...)

I should caution you that the PG dialect has significant differences from DuckDB, which is one of the reasons I made the Taco. I agree that launching from the command line on a Mac is a tad annoying, but with the Taco it is at least easy enough to add one argument. In fact, you can create a simple one line AppleScript to add the argument:

```
do shell script "\"/Applications/Tableau Desktop 2023.2.app/Contents/MacOS/Tableau\" -DDisableVerifyConnectorPluginSignature=true"
quit
```

I should add that to the documentation. I gather you can do something similar on Windows but it has been a while.

**Comment by erikhuck** [comment]
@hawkfish Perhaps we can also change the official documentation as that's where DuckDB and Tableau users will probably look first. I'm pretty sure the only difference between what I did and the [current documentation](https://duckdb.org/docs/guides/data_viewers/tableau.html) was I had to use a different file path for the JDBC driver. The documentation currently instructs to copy `duckdb_jdbc-osx-universal.jar` to `~/Library/JDBC` but I had to instead copy it to `~/Library/Tableau/Drivers/`. I think that simple change will do it for Mac users.
